torch/nn/quantized/modules/conv.py

```diff
@@ -16,10 +16,103 @@
 from torch.nn.quantized.modules.utils import _quantize_weight
 from torch.nn.utils import fuse_conv_bn_weights
 
+
+class ConvPackedParams(torch.nn.Module):
+    def __init__(self, spatial_dim):
```

Comment:
maybe make default value 2?

Reply:
I don't think a default value saves us much here, and it is bound to cause trouble when others are trying to use this class

Reply:
I see, makes sense